Assignment/Program.cs

```diff
@@ -1,7 +1,21 @@
 ﻿namespace Assignment {
    internal class Program {
       static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+         Console.Write ("Enter total votes : ");
+         string input = Console.ReadLine ()!.ToLower ().Replace (" ", "");
+         VotingContext (input);
+      }
+
+      public static void VotingContext (string input) {
+         Dictionary<char, int> vote = new ();
+         for (int i = 0; i < input.Length; i++) {
+            if (vote.ContainsKey (input[i]))

```

Comment:
if and else has only one statement you could write in the same line.

Reply:
Done